src/rexsdesktop/controller/General.java

```diff
@@ -238,7 +238,7 @@ public static String getEdicion() {
         Db db = new Db();

```

Comment:
you are repeating this in all of your method, it would be better if you inject this as a dependency of this class.